source/settings.txt

```diff
@@ -28,7 +28,7 @@ The following table lists the available settings for
 |vsce|:
 
 .. list-table::
-   :widths: 25 50 25
+   :widths: 25 45 30
```

Comment:
adjusted the widths of these table columns for better visibility of the 2 code block default examples